src/main/java/de/zalando/aruha/nakadi/validation/RFC3339DateTimeValidator.java

```diff
@@ -0,0 +1,39 @@
+package de.zalando.aruha.nakadi.validation;
+
+import org.everit.json.schema.FormatValidator;
+
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+
+public class RFC3339DateTimeValidator implements FormatValidator {
+
+    private final String errorMessage = "must be a valid date-time";
+
+    // Valid offsets are either Z or hh:mm. The format hh:mm:ss is not valid
+    private final String dateTimeOffsetPattern = "^.*(Z|((\\+|-)\\d\\d:\\d\\d))$";
```

Comment:
One should always remember this:
> Some people, when confronted with a problem, think 
> “I know, I'll use regular expressions.”   Now they have two problems.

The proper way would be to have a look how `ISO_OFFSET_DATE_TIME` is defined and derive proper format based on that:
```java
        DateTimeFormatter rfc3339 = new DateTimeFormatterBuilder()
            .parseCaseInsensitive()
            .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
            .appendOffset("+HH:MM", "Z")
            .toFormatter(Locale.getDefault(Locale.Category.FORMAT))
            .withChronology(IsoChronology.INSTANCE)
            .withResolverStyle(ResolverStyle.STRICT);
```